### ISSUE TITLE
Nuget : StartUpCommand

### ISSUE THREAD

**Issue by IncodingSoftware** [Incoding-Software/Incoding-Framework#164]
Hi @vgelmut 

What you think about pre-create StartUpCommand  ( with empty execute ) and then call it from Bootstrapper ? 

So, we can add some logic before starting application. Actually it will be cover previous ISetUp.

**Comment by vgelmut** [comment]
you mean derive custom project-specific StartUp commands from this empty one?

**Comment by IncodingSoftware** [comment]
Yes. Everyone project have some logic in start. StartUpCommand is It will be simple command which call other commands

**Comment by vgelmut** [comment]
So, just put that into template, so everyone can fill up? yes, good idea

**Comment by IncodingSoftware** [comment]
` so everyone can fill up`
of course. It is looks like are standard coding for logic in start application.
